system/settings/settings.xml

```diff
@@ -523,7 +523,7 @@
         </setting>
         <setting id="subtitles.height" type="integer" parent="subtitles.font" label="289" help="36186">
           <level>3</level>
-          <default>28</default>
+          <default>42</default>
```

Comment:
Yesterday when testing your branch on top of a previous installation I was presented with really small fontsizes. This happens because I had the default value of 28 in my guisettings.xml file. I think we should change the setting id (e.g. to subtitle.fontsize) to avoid similar issues. I have no strong opinions about the default value change for the alignment setting but I think the same applies, if the default value changes we should stop using the same Id for the setting.

Reply:
Actually the alignment one should be fine. You didn't change the order. The fontsize one is important though, the value interpretation now seems to be different

Reply:
yes the now it is different because the font is rendered with the right proportion between points/pixels
i will rename as suggested to `subtitle.fontsize`